scripts/Interface/ui/sans_isis/SANSSaveOtherWindow.py

```diff
@@ -0,0 +1,121 @@
+# Mantid Repository : https://github.com/mantidproject/mantid
+#
+# Copyright &copy; 2018 ISIS Rutherford Appleton Laboratory UKRI,
+#     NScD Oak Ridge National Laboratory, European Spallation Source
+#     & Institut Laue - Langevin
+# SPDX - License - Identifier: GPL - 3.0 +
+from . import ui_save_other_dialog as ui_save_other_dialog
+from PyQt4 import QtGui, QtCore
+from sans.common.enums import SaveType
+from mantidqtpython import MantidQt
+
+try:
+    from mantidplot import *
+    canMantidPlot = True
+except ImportError:
+    canMantidPlot = False
+
+
+class SANSSaveOtherDialog(QtGui.QDialog, ui_save_other_dialog.Ui_SaveOtherDialog):
+    def __init__(self):
+        super(QtGui.QDialog, self).__init__()
+        self.subscribers = []
+        self.setup_view()
+
+    def setup_view(self):
+        self.setupUi(self)
+        self.filename_lineEdit.textChanged.connect(self.on_file_name_changed)
+        self.browse_pushButton.pressed.connect(self.on_browse_clicked)
+        self.save_button.pressed.connect(self.on_save_clicked)
+        self.help_button.clicked.connect(self._on_help_button_clicked)
+        self.cancel_button.pressed.connect(self.on_cancel_clicked)
+        self.directory_lineEdit.textChanged.connect(self.on_directory_changed)
+        self.NxCanSAS_checkBox.setChecked(True)
```

Comment:
Lower-case variable name for consistency?

Reply:
done